WcaOnRails/app/views/competitions/edit.html.erb

```diff
@@ -122,6 +122,37 @@
         });
       </script>
 
+
+      <% if @competition_admin_view %>
```

Comment:
I'd like it too, but there's a problem. When creating a competiton, there is no date or location set yet, so nothing will appear inside the table. Is there a way to dinamically reload the table after location and date have been set for the current comp?